src/main/java/org/opensearch/index/codec/customcodecs/Lucene95CustomStoredFieldsFormat.java

```diff
@@ -27,9 +27,9 @@ public class Lucene95CustomStoredFieldsFormat extends StoredFieldsFormat {
     /** A key that we use to map to a mode */
     public static final String MODE_KEY = Lucene95CustomStoredFieldsFormat.class.getSimpleName() + ".mode";
 
-    private static final int ZSTD_BLOCK_LENGTH = 10 * 48 * 1024;
-    private static final int ZSTD_MAX_DOCS_PER_BLOCK = 4096;
-    private static final int ZSTD_BLOCK_SHIFT = 10;
+    public static final int ZSTD_BLOCK_LENGTH = 10 * 48 * 1024;
```

Comment:
@sarthakaggarwal97 why those are changed to public?

Reply:
These constants would be accessible to other plugins who are trying to instantiate custom StoredFieldsFormat and would prevent from deviating from custom-codecs package in the future.

Reply:
We can avoid this if you think we would be violating the principle of least access. I'm open to your suggestion

Reply:
I would imagine that the codec would be inherited from, right? We could make them protected but certainly not public

Reply:
updated, thanks @reta